types/office-js-preview/index.d.ts

```diff
@@ -23355,29 +23346,29 @@ declare namespace Excel {
          */
         getSpecialCellsOrNullObject(cellType: "ConditionalFormats" | "DataValidations" | "Blanks" | "Constants" | "Formulas" | "SameConditionalFormat" | "SameDataValidation" | "Visible", cellValueType?: "All" | "Errors" | "ErrorsLogical" | "ErrorsNumbers" | "ErrorsText" | "ErrorsLogicalNumber" | "ErrorsLogicalText" | "ErrorsNumberText" | "Logical" | "LogicalNumbers" | "LogicalText" | "LogicalNumbersText" | "Numbers" | "NumbersText" | "Text"): Excel.RangeAreas;
         /**
-         * Gets the range object containing the anchor cell for a cell getting spilled into. Fails if applied to a range with more than one cell. Read-only.
+         * Gets the range object containing the anchor cell for a cell getting spilled into. Fails if applied to a range with more than one cell.
          *
          * [Api set: ExcelApi BETA (PREVIEW ONLY)]
          * @beta
          */
         getSpillParent(): Excel.Range;
         /**
-         * Gets the range object containing the anchor cell for a cell getting spilled into. Read-only.
+         * Gets the range object containing the anchor cell for a cell getting spilled into.
                     If it is not a spill cell or more than once cells are give, a null object will be returned.
```

Comment:
Good eye. I need to circle back with the feature team and design council to review these Range Spill APIs. Are you okay with this issue persisting through this PR, since it was already present?

Reply:
Yes, that makes sense